### DISCUSSION TITLE
MainsMeter API

### DISCUSSION THREAD

**Discussion by nis** [serkri/SmartEVSE-3#157] [Q&A] [answered]
I'm having trouble with the API. I would like to send L1/2/3 as I have no smart meter attached to the EVSE.

Is this not how I should set mains_meter to API?
`response = r.post(server + '/settings?mains_meter=API')`
I get a None response.

If I add a 'override_current' I can set that without trouble:
`response = r.post(server + '/settings?override_current=60&mains_meter=API')`
I get a JSON response with only `"override_current": 60` back and the setting is changed.

**Comment by dingo35** [comment]
You should use the /currents endpoint:

https://swagger-ui.serkri.be/

**Reply by dingo35** [reply, marked as answer]
And mainsmeter is set at the LCD screen  not by API.

**Reply by nis** [reply]
Ooooh! That config!
Thank you :)
